crate/utils/src/crypto/wrap/encrypt_decrypt.rs

```diff
@@ -176,47 +167,37 @@ pub fn decrypt_bytes(
     Ok(plaintext)
 }
 
-#[cfg(feature = "fips")]
 fn decrypt_with_private_key(
     p_key: PKey<Private>,
     ciphertext: &[u8],
 ) -> Result<Vec<u8>, KmipUtilsError> {
-    // Unwrap symmetric key using RSA.
-    if p_key.rsa().is_err() {
-        kmip_utils_bail!("Decryption Error: Only RSA KEM supported in FIPS mode.")
+    if p_key.rsa().is_ok() {
```

Comment:
looks like we are still using `p_key` even if `is_ok()` is false. Since this code is not FIPS related anymore, can it be merge?

Reply:
Indeed, commit 7e5e9403330e8683c2bbddc9f5f4864a4bfac9ed should fix this.